packages-content-model/roosterjs-content-model-adapter/lib/editor/AdapterEditor.ts

```diff
@@ -130,6 +141,64 @@ export class AdapterEditor implements IEditor {
         return !this.core;
     }
 
+    //#region Content Model Editor members
+
+    /**
+     * Create Content Model from DOM tree in this editor
+     * @param option The option to customize the behavior of DOM to Content Model conversion
+     */
+    createContentModel(
+        option?: DomToModelOption,
+        selectionOverride?: DOMSelection
+    ): ContentModelDocument {
+        const core = this.getCore();
+
+        return core.api.createContentModel(core, option, selectionOverride);
+    }
+
+    /**
+     * Set content with content model
+     * @param model The content model to set
+     * @param option Additional options to customize the behavior of Content Model to DOM conversion
+     * @param onNodeCreated An optional callback that will be called when a DOM node is created
+     */
+    setContentModel(
+        model: ContentModelDocument,
+        option?: ModelToDomOption,
+        onNodeCreated?: OnNodeCreated
+    ): DOMSelection | null {
+        const core = this.getCore();
+
+        return core.api.setContentModel(core, model, option, onNodeCreated);
+    }
+
+    /**
+     * Get current running environment, such as if editor is running on Mac
+     */
+    getEnvironment(): EditorEnvironment {
+        return this.getCore().environment;
+    }
+
+    /**
+     * Get current DOM selection
+     */
+    getDOMSelection(): DOMSelection | null {
```

Comment:
Should stop using getSelectionRangeEx and select and use this function in content model editor?

Reply:
Yes, we should.